### ISSUE TITLE
Need option to show different size for placeholder

### ISSUE THREAD

**Issue by abhishektt** [tuupola/jquery_lazyload#274]
I've a loading image which is 64x64px and actual image size is 400x400 px. When I try to put my loading image as placeholder in src value, it gets stretched. It looks ugly. Can this be achieved or added as new feature?

**Comment by jsantana90** [comment]
have you found a way to solve this as I'm having the same issue, I have a loading.gif image which gets stretched because my background-size is set to `cover`.

**Comment by abhishektt** [comment]
Nope. I used div lazy load instead of this & it worked like charm.
